src/include/target/MATEK_2400_RX.h

```diff
@@ -1,5 +1,5 @@
 #ifndef DEVICE_NAME
-#define DEVICE_NAME "MATEK 2400RX"
```

Comment:
Inconsistent, should this not be `MATEK 2G4RX`

Reply:
As per Matek, these are called R24, so that's what I followed.
R24D - Diversity
R24S - SMD Antenna

R24P - PWM

The RXs aren't as critical as well, imho.